### DISCUSSION TITLE
Documentation about reverting local (cloud) changes

### DISCUSSION THREAD

**Discussion by BoswachterMarc** [opengisch/QField#2469] [Q&A]
Something in the documentation for QFieldCloud  use is not clear to  me.
According to the doc there is the possibility to 'revert local changes'. But the explanation says 'revert all modified features in the local **cloud** layers'. As I understand it is not about any cloud layers but it is about reverting (undo) the local changes in the QField app. If my interpretation is correct, I would skip the word **cloud**.

![afbeelding](https://user-images.githubusercontent.com/57868975/147101924-e1718970-4424-4805-865c-b78bb0702e12.png)

**Comment by suricactus** [comment]
FYI @marioba

**Comment by marioba** [comment]
In order to provide a clearer and more manageable discussion forum and to have all discussions about QField and related products (i.e. QFieldCloud and QFieldSync) in a single place, we transferred all the discussions from the QFieldCloud to the [QField repository](https://github.com/opengisch/QField/discussions).

**Comment by suricactus** [comment]
@BoswachterMarc thanks for the suggestion. PR opened with the proposed change https://github.com/opengisch/QField/pull/2573
